docs/reference/api/docker_remote_api_v1.14.md

```diff
@@ -126,7 +126,11 @@ Create a container
              "Tty":false,
              "OpenStdin":false,
              "StdinOnce":false,
-             "Env":null,
+             "Env": [
+                     "FOO=bar",
+                     "FOO=baz",
```

Comment:
Did you add a dup name on purpose?  Seems kind of like a funky thing to add in an example.